nautobot/ipam/views.py

```diff
@@ -424,7 +424,7 @@ class PrefixListView(generic.ObjectListView):
     filterset_form = forms.PrefixFilterForm
     table = tables.PrefixDetailTable
     template_name = "ipam/prefix_list.html"
-    queryset = Prefix.objects.all()
+    queryset = Prefix.objects.annotate(location_count=Count("locations"))
```

Comment:
We should be using `count_related` instead here and elsewhere in order to properly preserve queryset order.